### ISSUE TITLE
Join Error

### ISSUE THREAD

**Issue by LogGits** [VolmitSoftware/Mortar#6]
```
[12:30:05] [Server thread/ERROR]: Could not pass event PlayerJoinEvent to Mortar v1.0.56
java.util.NoSuchElementException: packet_handler
	at io.netty.channel.DefaultChannelPipeline.getContextOrDie(DefaultChannelPipeline.java:1119) ~[patched_1.13.2.jar:git-Paper-626]
	at io.netty.channel.DefaultChannelPipeline.addBefore(DefaultChannelPipeline.java:260) ~[patched_1.13.2.jar:git-Paper-626]
	at io.netty.channel.DefaultChannelPipeline.addBefore(DefaultChannelPipeline.java:249) ~[patched_1.13.2.jar:git-Paper-626]
	at com.comphenix.protocol.injector.netty.PipelineProxy.addBefore(PipelineProxy.java:66) ~[?:?]
	at com.comphenix.protocol.injector.netty.ChannelInjector$3$1.addBefore(ChannelInjector.java:272) ~[?:?]
	at mortar.api.nms.TinyProtocol.injectPlayer(TinyProtocol.java:51) ~[?:?]
	at mortar.api.nms.TinyProtocol.onPlayerJoin(TinyProtocol.java:101) ~[?:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor25.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.13.2.jar:git-Paper-626]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:76) ~[patched_1.13.2.jar:git-Paper-626]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:66) ~[patched_1.13.2.jar:git-Paper-626]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:520) ~[patched_1.13.2.jar:git-Paper-626]
	at net.minecraft.server.v1_13_R2.PlayerList.onPlayerJoin(PlayerList.java:369) ~[patched_1.13.2.jar:git-Paper-626]
	at net.minecraft.server.v1_13_R2.PlayerList.a(PlayerList.java:180) ~[patched_1.13.2.jar:git-Paper-626]
	at net.minecraft.server.v1_13_R2.LoginListener.b(LoginListener.java:169) ~[patched_1.13.2.jar:git-Paper-626]
	at net.minecraft.server.v1_13_R2.LoginListener.tick(LoginListener.java:65) ~[patched_1.13.2.jar:git-Paper-626]
	at net.minecraft.server.v1_13_R2.NetworkManager.a(NetworkManager.java:271) ~[patched_1.13.2.jar:git-Paper-626]
	at net.minecraft.server.v1_13_R2.ServerConnection.c(ServerConnection.java:129) ~[patched_1.13.2.jar:git-Paper-626]
	at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:1132) ~[patched_1.13.2.jar:git-Paper-626]
	at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:439) ~[patched_1.13.2.jar:git-Paper-626]
	at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:940) ~[patched_1.13.2.jar:git-Paper-626]
	at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:837) ~[patched_1.13.2.jar:git-Paper-626]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_201]
```

**Comment by DavidRockin** [comment]
Duplicate of https://github.com/VolmitSoftware/Gloss/issues/71

**Comment by DavidRockin** [comment]
should be fixed in 45918a95ed2ba15fc46e988317ec396a1100ace8
